waspc/src/Wasp/Generator/SdkGenerator/Auth/AuthFormsG.hs

```diff
@@ -20,6 +20,8 @@ genAuthForms auth =
   sequence
     [ genAuthComponent auth,
       genTypes auth,
+      -- todo (move this to somewhere more meaningful)
+      genFileCopy [relfile|core/stitches.config.ts|],
```

Comment:
I've left this comment here because we probably want to move this file outside of `core`.